src/LibChorus/VcsDrivers/Mercurial/HgRepository.cs

```diff
@@ -1475,10 +1471,28 @@ private static void SetExtensions(IniDocument doc, IEnumerable<KeyValuePair<stri
 			}
 		}
 
-		internal static bool CheckExtensions(IniDocument doc, IEnumerable<KeyValuePair<string, string>> extensionDeclarations)
+		/// <summary/>
+		/// <returns>
+		/// Returns true if the extensions in the repo contain only the extensions declared in the given map.
+		///</returns>
+		internal static bool CheckExtensions(IniDocument doc, Dictionary<string, string> extensionDeclarations)
```

Comment:
I suppose this extensions file we're messing with is private to Chorus? We wouldn't want to mess up the set of extensions the user is using to manage something else.
How important is it that no other extensions are active? Hg will, I believe, look for a global settings file which might also configure extensions. Or do we do something to suppress that? I'm wondering whether we need to do something here to explicitly DISable any known extensions that cause problems.